app/frontend/src/application.js

```diff
@@ -2,6 +2,7 @@ import { Application } from '@hotwired/stimulus';
 
 import PanelController from '../../components/panel_component/panel';
 import UtilsController from './components/utils';
+import AutoCompleteController from './components/autocomplete/autocomplete';
```

Comment:
shouldn't this be `AutocompleteController` ?

Reply:
whilst it doesnt actually matter can call it what we want, in keeping with the filename/folder name you are right